test/unit-test/multi/test-continuable-await.cpp

```diff
@@ -169,6 +169,53 @@ TYPED_TEST(single_dimension_tests, are_awaitable_with_cancellation_from_coro) {
   ASSERT_ASYNC_CANCELLATION(resolve_coro_canceled(supply))
 }
 
+template <typename S>
+cti::continuable<> test_symmetric_transfer(S&& supplier) {
+  // If symmetric transfer is not working properly, large
+  // loops will quickly cause stack overflows.
+  for (size_t index = 0; index < 10000; index++) {
+    co_await supplier();
+  }
+}
```

Comment:
co_return is missing here

Reply:
IIRC the co_return is optional, similar to a regular return, but it won't hurt. Added it.